styleguide/components/41-organisms/programme/_programme.scss

```diff
@@ -0,0 +1,194 @@
+.programme {
+  width: calc(100% + 2 * #{$gutter-width});
+  margin-right: -$gutter-width;
+  margin-left: -$gutter-width;
+
+  > .inner-box {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+
+    &::before {
+      @include spot-image('calendar');
+
+      @include tablet {
+        right: 0;
+        left: 0;
+      }
+    }
+  }
+
+  ///
+  /// Programme slot.
+  ///
+  &-slot {
+    max-width: $bp-max-content;
+    margin: 0 auto;
+
+    &-title {
+      margin: 0 auto 1.6rem;
+      font-size: 1.3rem;
+      text-align: center;
+    }
+
+    > dd {
+      max-width: none;
+      padding: 0;
+    }
+
+    & + & {
```

Comment:
This is a bid obscure don't you think? It's okay for me but maybe add a comment for people who are not super familiar with SASS? :)

Reply:
I am not afraid to admit I had to look it up. :wink:

Reply:
Don't know.. ? It's just the combination of: 

- The `+` selector in CSS, which is possibly not known by every developer but it should be. We're using the `>` selector as well?
- the `&`-sign in SASS, which is widely used throughout all SASS files in the repo.

I do acknowledge that it could be a weird construct if you see it for the first time, but I've seen it in numerous projects and use it a lot myself.

Wdyt? Keep it or throw it out?

Reply:
Keep it, no need to comment for me either.